recipe/meta.yaml

```diff
@@ -1,15 +1,16 @@
 {% set name = "r-tiledb" %}
-{% set version = "0.26.0" %}
+{% set version = "0.27.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/TileDB-Inc/TileDB-R/archive/{{ version }}.tar.gz
-  sha256: fea1c665c3fb55dfd02b09eb59ed1f7e02bb07c26ec5e56db53028f9bb9074ef
-  patches:   # [osx and not arm64]
+  sha256: 0ccf6726274655115ba04378d8aefd0c057867d6725144d9e5dc9522c914cfd0
+  patches:
     - 0001-skip-osx-test-to-avoid-spurious-segfault.patch  # [osx and not arm64]
+    - 0002-skip-osx-test-to-avoid-spurious-segfault.patch  # [linux and aarch64]
```

Comment:
```suggestion
    - 0002-skip-linux-aarch64-test-to-avoid-spurious-fail.patch  # [linux and aarch64]
```

Reply:
🤕 that was silly and embarrassing on my part -- @jdblischak thanks for the catch :|

Reply:
We've all been there before, especially me! These are the exact types of problems that require a second set of fresh eyes